Source/KakapoServer.swift

```diff
@@ -74,19 +73,19 @@ class KakapoServer: NSURLProtocol {
         
     }
     
-    static func get(urlString: String, handler: (request: KakapoRequest) -> ()) {
+    static func get(urlString: String, handler: (request: Request) -> ()) {
         KakapoServer.routes[urlString] = (.GET, handler)
     }
     
-    static func post(urlString: String, handler: (request: KakapoRequest) -> ()) {
+    static func post(urlString: String, handler: (request: Request) -> ()) {
         KakapoServer.routes[urlString] = (.POST, handler)
     }
     
-    static func del(urlString: String, handler: (request: KakapoRequest) -> ()) {
+    static func del(urlString: String, handler: (request: Request) -> ()) {
```

Comment:
Still not related but I would use `delete` instead of `del`. Keeps it clear and matches with the HTTP method name.